src/AppRouter.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './model/store';
 import App from './App';
 
 const AppRouter = () => (
-  <Router>
-    <>
-      <Route
-        exact
-        path="/"
-        component={({ history, match }) => (
-          <App history={history} initialState={match.params} />
-        )}
-      />
-    </>
-  </Router>
+  <Provider store={store}>
+    <Router>
+      <>
+        <Route
+          exact
+          path="/"
+          component={({ history, match }) => (
+            <App history={history} initialState={match.params} />
+          )}
+        />
+      </>
+    </Router>
+  </Provider>
```

Comment:
If you move the provider here, you ll not be able to use the App component properly in the api because you'll not have the store set. I think that's why it was in the App and not here.